### PR TITLE
dns/ddclient: add native service Domeneshop

### PR DESCRIPTION
**Pull request by bernhardfrenking**
Add **[Domeneshop](https://domene.shop/)** as new service to update DNS records via the native ddclient plugin.
The API documentation can be found [here](https://api.domeneshop.no/docs/#tag/ddns/paths/~1dyndns~1update/get).

### CODE REVIEW

File: dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf [hunk 1]
```diff
@@ -53,6 +53,8 @@ zone={{account.zone}}, \
 {%      elif account.service == 'dns-o-matic' %}
 protocol=dyndns2, \
 server=updates.dnsomatic.com, \
+{%      elif account.service == 'domeneshop' %}
+protocol=domeneshop, \
```
Comment (AdSchellevis) [inline dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf:57]: when it's only available in the native service, you shouldn't need a ddclient config.

Comment (bernhardfrenking) [reply dns/ddclient/src/opnsense/service/templates/OPNsense/ddclient/ddclient.conf:57]: OK, then I misunderstood the ddclient.conf.

File: dns/ddclient/src/opnsense/scripts/ddclient/lib/account/domeneshop.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+                )
+            else:
+                syslog.syslog(
+                    syslog.LOG_ERR,
+                    "Account %s failed to set new ip %s [%d - %s] for %s" % (
+                        self.description, self.current_address, response.status_code, response.text.replace('\n', ''), hostnames
+                    )
+                )
+
+        return False
```
Comment (fichtner) [inline dns/ddclient/src/opnsense/scripts/ddclient/lib/account/domeneshop.py:93]:
```suggestion
        return False

```

**Comment by bernhardfrenking** [comment]
@fichtner & @AdSchellevis: Is it ready for merging?

**Comment by AdSchellevis** [comment]
@bernhardfrenking it looks like it, thanks!

**Pull request merged**